src/main/java/com/example/pokebattlez/battle/controller/BattleController.java

```diff
@@ -15,8 +16,18 @@ public BattleController(BattleService battleService) {
         this.battleService = battleService;
     }
 
-    @MessageMapping("/move")
-    public void chooseMove(@RequestParam("id") Long playerId, @RequestParam("index") Integer index) {
+    @MessageMapping("/{battleId}/move")
+    public void chooseMove(@DestinationVariable String battleId, @RequestParam("id") Long playerId, @RequestParam("index") Integer index) {
```

Comment:
I guess at some point the playerId will be encoded somehow, so as not to be able to move in somebody else's stead?

Reply:
Absolutely, we are gonna use JWT for identifying connections.